src/note/note.service.ts

```diff
@@ -0,0 +1,32 @@
+import { Injectable, Logger } from '@nestjs/common';
+import {InjectRepository } from '@nestjs/typeorm';
+import { Repository } from 'typeorm';
+
+import { notes } from './note.entity';
+import { Note } from '../interfaces/note'
+@Injectable()
+export class  NoteService {
+constructor(
+@InjectRepository(notes)
+private noteRepository: Repository<notes>,
+) {}
+async index(): Promise<Note[]> {
```

Comment:
nazwa do zmiany, index() niestety nic nie mówi, dobra nazwa to np. getAll()

Reply:
Poprawione. Kiedyś majstrowałem przy larawelu i był tam generator controllerów i była tam właśnie taka metoda jak index z tond mi to zostało hehe